src/feature_extractor.h

```diff
@@ -1,9 +1,13 @@
+// -*-c++-*-
+
 #ifndef FEATURE_EXTRACTOR_H
 #define FEATURE_EXTRACTOR_H
 
 #include <rcsc/player/player_agent.h>
 #include <vector>
 
+class Agent;
```

Comment:
Oh. Yes, that's left-over from putting Agent in the feature extractor. Will remove. Oops!